runtime/src/bank.rs

```diff
@@ -4073,8 +4073,7 @@ impl Bank {
         timings: &mut ExecuteTimings,
     ) -> LoadAndExecuteTransactionsOutput {
         let sanitized_txs = batch.sanitized_transactions();
-        debug!("processing transactions: {}", sanitized_txs.len());
-        inc_new_counter_info!("bank-process_transactions", sanitized_txs.len());
+        inc_new_counter_debug!("bank-process_transactions", sanitized_txs.len());
```

Comment:
we rely on this metric for transactions per second monitoring: 
<img width="1197" alt="image" src="https://user-images.githubusercontent.com/3374799/163850200-c8abed57-ea65-4f24-89a4-06ab04af377d.png">

so this will break that. A fix would probably be to aggregate these as well